Client.xcodeproj/project.pbxproj

```diff
@@ -9703,12 +9705,7 @@
 			isa = PBXGroup;
 			children = (
 				8A171A6029F82AD90085770E /* Application */,
-				8A93F86329D37314004159D9 /* Coordinators */,
 				96CB5D5F29B22E860034AE0A /* Autofill */,
-				5A70EF17295E2DF400790249 /* DependencyInjection */,
-				8ACA8F722919870B00D3075D /* Helpers */,
-				8A96C4B728F9DD0600B75884 /* Extensions */,
-				8AED868428CA7B1200351A50 /* TabManagement */,
 				CA7BD564248185B500A0A61B /* BreachAlertsTests.swift */,
```

Comment:
There's deletion in the `pbxproj` that shouldn't be there I believe

Reply:
Ya true. Not sure how that happened. Any tips on reverting this change?

Reply:
Since the PR and changes are small, I would maybe just recreate a new branch rather than tinkering with git (unless you wish to, but I try to not mess too much with the `pbxproj` as things can go awry quickly). 
- Make sure you are based of latest `main`, currently 1d10840812464563033d7b52fd008175731ed7d3
- Recreate the `DownloadTests.swift` file, and make the change again in `DownloadQueue.swift`. Make sure you only create the file, and no folders/files are deleted in the process
- You can check locally before commit what your git diff includes, it should just add the new DownloadTests file in the `pbxproj`
- Open a new PR and we close this one 🤷‍♀️